cmd/aasrepositoryservice/Dockerfile

```
@@ -0,0 +1,39 @@
+# ---------- Build stage ----------
+FROM golang:1.25-alpine3.22 AS builder
+
+WORKDIR /app
+
+# Install git (needed for go modules in some environments)
+RUN apk add --no-cache git
+
+# Copy go mod files first for caching
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy the rest of the source
+COPY . .
+
+# Build the AAS repository service
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
+    go build -o aasrepositoryservice ./cmd/aasrepositoryservice
+
+
+# ---------- Runtime stage ----------
+FROM alpine:3.19
```

Please use alpine 3.22 here aswell
```suggestion
FROM alpine:3.22
```